app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def formatted_time
```

Comment:
we'll have to make sure our merge is okay and decide whichever formatted time method we want